pkg/runtime/runtime.go

```diff
@@ -2494,7 +2494,8 @@ func (a *DaprRuntime) processComponentSecrets(component componentsV1alpha1.Compo
 
 		resp, ok := cache[m.SecretKeyRef.Name]
 		if !ok {
-			r, err := secretStore.GetSecret(secretstores.GetSecretRequest{
+			// TODO: cascade context.
+			r, err := secretStore.GetSecret(context.TODO(), secretstores.GetSecretRequest{
```

Comment:
Why not `context.Background()` ?

Reply:
```suggestion
			r, err := secretStore.GetSecret(context.Background(), secretstores.GetSecretRequest{
```

Reply:
`context.TODO` iscorrect here as it's a placeholder for cascading context in the future